app/models/validations/setup_validations.rb

```diff
@@ -150,7 +150,7 @@ def add_same_merge_organisation_error(record)
     elsif absorbing_owning_organisation_inactive?(record)
       record.errors.add :startdate, I18n.t("validations.setup.startdate.invalid_absorbing_organisations_start_date.same_organisation",
                                            owning_organisation: record.owning_organisation.name,
-                                           owning_organisation_available_from: record.owning_organisation.created_at.to_formatted_s(:govuk_date))
+                                           owning_organisation_available_from: record.owning_organisation.available_from.to_formatted_s(:govuk_date))
```

Comment:
is there any risk of available_from not existing here?

Reply:
`absorbing_owning_organisation_inactive?` checks that `record.owning_organisation.available_from.present?`, sos should be all good!